charts/manager/templates/manager/feature-flag.yaml

```
@@ -7,12 +7,20 @@ data:
     ################################
     #                              #
     #    EXAMPLE CONFIGURATION     #
-    #                              # 
-    ################################ 
-    # enable dex features 
-    # storage allows you to choose what kind of dex storage backend you would like to choose 
+    #                              #
+    ################################
+    # enable dex features
+    # storage allows you to choose what kind of dex storage backend you would like to choose
     # Greenhouse gives the choice of using kubernetes as the storage backend or to use postgresSQL as the backend
     dex: |
       storage: kubernetes / postgres
+    # enable template features
+    # rendering allows you to enable or disable template rendering for plugin option values
+    # set to false to disable template rendering (templates will be used as literal values)
+    # set to true to enable template rendering (templates will be evaluated)
+    template: |
+      rendering: false / true
```

Maybe we can rename the flag as such, so that the flag itself tells that it is related to plugin etc.
```suggestion
    plugin: |
       optionValueTemplating: false / true
```